package com.ibizabroker.bibliotheque.entity;

public class ReservationRequest {

    private Integer livreId;

    private Integer adherentId;

    public Integer getLivreId() {
        return livreId;
    }

    public void setLivreId(Integer livreId) {
        this.livreId = livreId;
    }

    public Integer getAdherentId() {
        return adherentId;
    }

    public void setAdherentId(Integer adherentId) {
        this.adherentId = adherentId;
    }
}
